esx_service/vmdk_ops.py

```diff
@@ -1183,6 +1188,7 @@ def setStatusAttached(vmdk_path, vm, vm_dev_info=None):
     if not vol_meta:
         vol_meta = {}
     vol_meta[kv.STATUS] = kv.ATTACHED
+    # instanceUuid i.e. vc uuid of a VM is stored in volume kv
```

Comment:
yeah, I recall there were cases with empty instanceUuid, though I think it was with older(er) ESX versions